lib/assert.js

```diff
@@ -807,11 +869,26 @@ function strict(...args) {
   innerOk(strict, args.length, ...args);
 }
 
+const assertInstance = new Assert({ diff: 'simple', strict: false });
+[
+  'ok', 'fail', 'equal', 'notEqual', 'deepEqual', 'notDeepEqual',
+  'deepStrictEqual', 'notDeepStrictEqual', 'strictEqual',
+  'notStrictEqual', 'partialDeepStrictEqual', 'match', 'doesNotMatch',
+  'throws', 'rejects', 'doesNotThrow', 'doesNotReject', 'ifError',
+].forEach((name) => {
+  const bound = assertInstance[name].bind(assertInstance);
+  ObjectDefineProperty(bound, 'name', { __proto__: null, value: name });
+  assert[name] = bound;
+});
```

Comment:
Have we considered changing the prototype instead? That would avoid all the bound calls.
```suggestion
ObjectSetPrototypeOf(assert, Assert.prototype);
```

Reply:
That would be a breaking change.

Reply:
Yeah it's not that great of an idea, it breaks our ESM integration (because we only expose own property as named exports), and there is code out there doing e.g. `assert.apply(…)`, so we would need to pass those along anyway. Please disregard.